packages/components/src/root-child/test/index.jsx

```diff
@@ -43,27 +35,28 @@ describe( 'RootChild', () => {
 
 	describe( 'rendering', () => {
 		test( 'should render any children as descendants of body', () => {
-			const tree = ReactDom.render( createElement( Greeting ), container );
+			render( <Greeting />, { container } );
 
-			expect( tree.parentChildRef.current.parentNode.className ).toBe( 'parent' );
-			expect( tree.rootChildRef.current.parentNode.parentNode ).toBe( document.body );
+			const wrapper = screen.getByTestId( 'parent-child' ).parentNode;
+
+			expect( wrapper ).toHaveClass( 'parent' );
+			expect( wrapper.parentNode ).toBe( container );
 		} );
 
 		test( 'should update the children if parent is re-rendered', () => {
-			const tree = mount( createElement( Greeting ), { attachTo: container } );
-			tree.setProps( { toWhom: 'Universe' } );
+			render( <Greeting toWhom="Universe" />, { container } );
```

Comment:
Is this testing re-render, or just the initial render?

Reply:
Good find, fixed the test to test rerendering in 4cb43407d166f00f483287a5e6775f4a236d74c9 👍